### PR TITLE
Documentation for dev/setup (old)

### PR DESCRIPTION
**Pull request by joeflack4**
- Updates: README.md: (i) reorganized/added 'dev' section, (ii) added project setup steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,7 +215,17 @@ Same using pronto, fetching ontology from obolibrary
 runoak -i obolibrary:go.obo  viz GO:0005773
 ```
 
-## Potential Refactoring
+## Developer notes
+### Local project setup
+Prerequisites:
+1. Python 3.10+
```
Comment (cmungall) [inline README.md:221]: 3.9 works

In fact if you look at the checks we do
https://github.com/INCATools/ontology-access-kit/blob/main/.github/workflows/main.yaml

The *only* guarantee we have is for 3.9

Comment (joeflack4) [reply README.md:221]: Thanks! This was just a guess on my part. will change it now

**Comment by joeflack4** [comment]
Looks like I still can't push to this trunk repository directly. I was looking at the [team permissions](https://github.com/orgs/INCATools/teams/oak-developers/repositories), and it looks like it only gives 'Read' access..

File: README.md [hunk 2]
```diff
@@ -215,7 +215,17 @@ Same using pronto, fetching ontology from obolibrary
...
+2. [Poetry](https://python-poetry.org/)
+
+Setup steps:
+1. Clone this repository.
```
Comment (cthoyt) [inline README.md:225]: would be better to explicitly write the commands in a shell-like format, including the git clone command, the cd, etc

See https://github.com/biopragmatics/bioregistry#-installation for an example

Comment (joeflack4) [reply README.md:225]: Not a bad suggestion. Done~

I did leave off the `$` though. Even though it's a common convention, I have some beef with that: (i) makes copy/paste more tedious, (ii) the code block formatting already helps with indication, (iii) not every prompt uses `$`.

**Comment by joeflack4** [comment]
Replacing this with #162

**Pull request closed without merging**